### PR TITLE
About myself and portfolio projects

### PR DESCRIPTION
**Pull request by Sajeelzafar**
I have prepared a mobile version of the given section.

The template is copied from Figma, with the exact code of CSS as mentioned there, so please don't count that as plagiarism.

Also, the dimensions and spaces were fixed in Figma, so the mobile version works best for the size of the Figma mobile version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,5 +74,105 @@ <h3>Skills</h3>
...
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
+            <button type="button" class="projectButton">See Project</button>
+        </div>
+
+
+    </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] Please the container of the section is in a `section` tag, kindly add your container to the `section` tag 🙃 🙃

File: index.html [hunk 2]
```diff
@@ -74,5 +74,105 @@ <h3>Skills</h3>
...
+            <div class="sidebar">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] The list of the project technologies is a list `ul` and each of them a `li`, please kindly fix it 🙃😁

File: index.html [hunk 3]
```diff
@@ -74,5 +74,105 @@ <h3>Skills</h3>
...
+            <div class="rightsidebar">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] The list of the project's technologies is a list `ul` and each of them a `li`, please kindly fix it 😁

File: index.html [hunk 4]
```diff
@@ -74,5 +74,105 @@ <h3>Skills</h3>
...
+            <div class="thirdparagraph">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] The list of the project's technologies is a list ul and each of them a li, please kindly fix it 😁

File: index.html [hunk 5]
```diff
@@ -74,5 +74,105 @@ <h3>Skills</h3>
...
+            <div class="fourthparagraph">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] The list of the project's technologies is a list ul and each of them a li, please kindly fix it 😁

File: style.css [hunk 6]
```diff
@@ -139,18 +139,159 @@
...
+    margin: 80px 24px 33px 24px;
+  }
+
+  .workGridContainer {
+    width: 354px;
+    margin: 40px 24px 0 24px;
+    display: grid;
+    grid-template-columns: auto;
     justify-content: center;
+  }
```
Comment (cynthiainga) [inline style.css]: - [ ] There is a strange margin to the right of your content that is not present in the template design, please kindly fix it 🙃 🙃

Your design
Screen Shot 2022-08-19 at 13 57 46

Figma design
Screen Shot 2022-08-19 at 13 56 27

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Sajeelzafar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] No linters ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Used correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly write your commit message in imperative tense and capitalized. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly 😁 😁 .

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
## Hi @Sajeelzafar   👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description
✔️  Good commit message 
✔️ Good use of Github flow 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] I would suggest you use good commit messages since their very descriptive while tracking the changes in your project 😁 
- [ ] Kindly make your pull request more descriptive so as to let us know what you implemented in the current branch 😀 
- [ ] kindly make sure the content of your all page has the max of 100% so that it will contain all screens and remove the scrolling button, check below the responsiveness of your page compare to the Figma design 


image
### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sajeelzafar** [comment]
Thank you @MaganaAsiati for taking out time from your busy schedule to go through my project. I'll surely keep into account the suggestions made by you, regarding commit messages, in my upcoming projects.

The third suggestion is what I am struggling with. I can clearly see that the website is not very responsive, as pointed out by you as well. On Figma, I am unable to see how it can be resized, I just see only one particular size. The distances are fixed, as given in pixels, and the background ellipses are "position: absolute".

Can you please elaborate a bit more on "content of your all page has the max of 100%" as I really want the design to be responsive? 

I would be looking forward to your reply.

**Pull request merged**